README.txt

```diff
@@ -0,0 +1 @@
+GRAPEFRUIT(resized): http://images.all-free-download.com/images/graphicthumb/grapefruit_clip_art_25749.jpg
```

Comment:
This should be inside `README.md` instead please. But also see the note inside the GCI task on this topic :-)